src/components/Filter.tsx

```diff
@@ -0,0 +1,62 @@
+import { useMemo } from 'react';
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+import { Todo } from '../utils/types/type';
+
+type Props = {
+  todos: Todo [],
+  setTodos: (todos: Todo[]) => void,
+};
+
+export const Filter:React.FC<Props> = ({ todos, setTodos }) => {
+  const itemsLeft = useMemo(() => todos
+    .filter(todo => !todo.completed).length, [todos]);
+  const hasComleatedItems = todos.length > itemsLeft;
```

Comment:
There is a typo in variable name `hasComleatedItems`, it should be `hasCompletedItems`.